diff --git a/drivers/spi/spi-pl022-starfive.c b/drivers/spi/spi-pl022-starfive.c
index 755ed9c6d51f..f0602cb77e01 100644
--- a/drivers/spi/spi-pl022-starfive.c
+++ b/drivers/spi/spi-pl022-starfive.c
@@ -2735,7 +2735,7 @@ static struct platform_driver starfive_of_pl022_driver = {
 	.remove = starfive_of_pl022_remove,
 };
 
-module_platform_driver(starfive_of_pl022_driver);
+// module_platform_driver(starfive_of_pl022_driver);
 /* platform register end */
 
 MODULE_AUTHOR("xingyu.wu <xingyu.wu@starfivetech.com>");
