From 6d118ed876e89180500984bb6dcaead0f7c6002f Mon Sep 17 00:00:00 2001
From: Raito Bezarius <masterancpp@gmail.com>
Date: Fri, 10 Feb 2023 14:09:30 +0100
Subject: [PATCH 6/7] media/starfive: kill some modules

---
 drivers/media/platform/starfive/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/starfive/Makefile b/drivers/media/platform/starfive/Makefile
index a1d46ed198bc..5469b1930963 100644
--- a/drivers/media/platform/starfive/Makefile
+++ b/drivers/media/platform/starfive/Makefile
@@ -8,11 +8,11 @@
 #obj-$(CONFIG_VIN_SENSOR_OV4689) += ov4689_mipi.o
 #obj-$(CONFIG_VIDEO_STF_VIN) += stf_vin.o stf_event.o stf_isp.o stf_csi.o
 
-obj-$(CONFIG_VIN_SENSOR_OV5640) += v4l2_driver/ov5640.o
+# obj-$(CONFIG_VIN_SENSOR_OV5640) += v4l2_driver/ov5640.o
 obj-$(CONFIG_VIN_SENSOR_SC2235) += v4l2_driver/sc2235.o
 obj-$(CONFIG_VIN_SENSOR_OV4689) += v4l2_driver/ov4689_mipi.o
 obj-$(CONFIG_VIN_SENSOR_OV13850) += v4l2_driver/ov13850_mipi.o
-obj-$(CONFIG_VIN_SENSOR_IMX219) += v4l2_driver/imx219_mipi.o
+# obj-$(CONFIG_VIN_SENSOR_IMX219) += v4l2_driver/imx219_mipi.o
 
 starfivecamss-objs += 	v4l2_driver/stfcamss.o \
 	                       	v4l2_driver/stf_event.o \
-- 
2.39.1

